rednotebook/external/txt2tags.py

```diff
@@ -4646,8 +4656,10 @@ def get_tagged_link(label, url):
     # Set link type
     if regex['email'].match(url):
         linktype = 'email'
+    elif regex['entryReference'].match(url):
+        linktype = 'entryReference'
     else:
-        linktype = 'url';
+        linktype = 'url'
 
     # Escape specials from TEXT parts
     label = doEscape(target,label)
```

Comment:
Instead of changing txt2tags.py (which will be difficult to maintain), I suggest to extend the language in the file utils/markup.py similarly to the hashtag and linebreak markup (which RedNotebook adds to the txt2tags language).